### PR TITLE
chore: catch `updateMetaInfo` error when init database

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

File: packages/neuron-wallet/src/startup/sync-block-task/init-database.ts [hunk 1]
```diff
@@ -9,8 +9,13 @@ export const initDatabase = async () => {
   try {
     const hash = await genesisBlockHash()
     await initConnection(hash)
-    const systemScriptInfo = await LockUtils.systemScript()
-    updateMetaInfo({ genesisBlockHash: hash, systemScriptInfo })
+
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/startup/sync-block-task/init-database.ts]: It seems to be nested in a try block.

Comment (classicalliu) [reply packages/neuron-wallet/src/startup/sync-block-task/init-database.ts]: Yes, get systemScriptInfo failed should not bother initConnection.
But if `initConnection` failed, should not execute those codes.

Comment (Keith-CY) [reply packages/neuron-wallet/src/startup/sync-block-task/init-database.ts]: If it is possible to run these two try blocks sequentially.

Comment (classicalliu) [reply packages/neuron-wallet/src/startup/sync-block-task/init-database.ts]: > But if `initConnection` failed, should not execute those codes.

If in two try blocks, no matter `initConnection` success or not, the second try block always be executed.

**Pull request merged**